src/components/themed/MenuTab.js

```diff
@@ -35,13 +35,13 @@ export class MenuTabComponent extends React.PureComponent<Props> {
 
     switch (route) {
       case 'walletList':
-        return Actions.jump('walletListScene')
+        return navigation.navigate('walletListScene')
       case 'pluginListBuy':
-        return Actions.jump('pluginListBuy', { direction: 'buy' })
+        return navigation.navigate('pluginListBuy', { direction: 'buy' })
       case 'pluginListSell':
-        return Actions.jump('pluginListSell', { direction: 'sell' })
+        return navigation.navigate('pluginListSell', { direction: 'sell' })
       case 'exchange':
-        return Actions.jump('exchange')
+        return navigation.navigate('exchange')
```

Comment:
This is a good change but it already exists in another PR so it might cause conflicts.
So I'm sorry to ask but can you remove this commit? It would get fixed anyway.